src/Uno.UI.Runtime.Skia.Gtk/buildTransitive/Uno.UI.Runtime.Skia.Gtk.targets

```diff
@@ -1,21 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <PropertyGroup>
-  </PropertyGroup>
-  
-  <Target Name="_UnoSkiaGtkFeatureDefines"
-		  BeforeTargets="PrepareForBuild">
+	<Target Name="_UnoSkiaGtkFeatureDefines" AfterTargets="PrepareForBuild">
 
-	<PropertyGroup>
-	  <UnoDefineConstants>$(UnoDefineConstants);UNO_REFERENCE_API;HAS_UNO_SKIA;HAS_UNO_SKIA_GTK</UnoDefineConstants>
-	</PropertyGroup>
+		<PropertyGroup>
+			<DefineConstants>$(DefineConstants);UNO_REFERENCE_API;HAS_UNO_SKIA;HAS_UNO_SKIA_GTK</DefineConstants>
```

Comment:
This is supposed to make a task-local version of DefineConstants. Are you sure this works?

Reply:
As far as I know, there are no task/target local properties in MSBuild at the moment. All properties are global. So constant duplication occurs because two targets use the same property, assuming it is empty
https://github.com/dotnet/msbuild/issues/2480

Reply:
There is also a setting that allows you to control visibility. But again, in the direction of limitation. Though that would also fix the duplication of constants:
TreatAsLocalProperty
https://docs.microsoft.com/en-us/visualstudio/msbuild/project-element-msbuild?view=vs-2022

Reply:
Looks like I got the wrong assumptions there about how properties behave, thanks for the links. This will do fine then!